.github/workflows/windows-build-and-test.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
       run: python .github/gh_config_reader.py
     - name: Set build_type
       id: build_type
+      env:
+        WSLENV: GITHUB_OUTPUT/p
```

Comment:
For the other actions that use the environment variable `GITHUB_OUTPUT`, we also set `shell: wsl-bash {0}`. Is this intentionally not set here?

Reply:
This is the config action that runs on ubuntu-latest, so bash is the default shell.

Reply:
I guess I have to remove the WSLENV...